CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 4.2.0
+
+This is a major Teleport release with a focus on new features and bug fixes.
+
+### Improvements
+
+* Alpha: Enhanced Session Recording let's you know what's really happening during a Teleport Session. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/features/enhanced_session_recording/).
+* Alpha: Workflows API lets admins escalate RBAC roles. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/enterprise/#approval-workflows).
+* Beta: Teleport provides HA Support using Firestone and Google Cloud Storage using Google Cloud Platform. [Read the docs](https://gravitational.com/teleport/docs/ver/4.2/gcp_guide/).
```

Comment:
Shall we include GH ticket #s on these?  I can probably chase them down with @benarent's help.

Reply:
Yes, absolutely.

Reply:
The issue numbers are here https://github.com/gravitational/teleport/issues/3133

Reply:
Included issue links for all 3 of these in f62c4620dd565a3894a900ab435f11a991e5e9e6